apps/vpnmanager/src/pages/index.tsx

```diff
@@ -1,10 +1,20 @@
-export default function Home() {
-  return null;
+import Statistics from "@/vpnmanager/components/Statistics";
+import { Data } from "@/vpnmanager/components/Statistics/Statistics";
+import { getStats } from "@/vpnmanager/lib/statistics";
+
+interface Props {
+  data: Data[];
+}
+export default function Home(props: Props) {
+  const { data } = props;
+  return <Statistics data={data} />;
 }
 
 export async function getStaticProps() {
+  const data = await getStats({ query: { orderBy: "date DESC" } } as any);
```

Comment:
`as any` is a code smell, let's not use it for areas where we have full control. I see that the issue is caused by declaring the argument in `getStats` to be of type `NextRequest`, let's instead narrow the value there to fit this invocation and get rid of the `as any`. I've left a comment there on my suggestion